.github/workflows/ci.yml

```diff
@@ -16,14 +16,13 @@ on:
       - reopened
       - synchronize
 
-defaults:
-  run:
-    shell: bash
-
 concurrency:
   group: ${{ github.ref_name }}-ci
   cancel-in-progress: true
 
+permissions:
+  contents: read
```

Comment:
@lucacome Are more permissions required for the build/release steps?

Reply:
yeah more permissions are required, we should change this in a different PR